DownloadManager/Plugin.DownloadManager.Android/DownloadFileImplementation.cs

```diff
@@ -92,20 +94,44 @@ public DownloadFileImplementation (ICursor cursor)
             Url = cursor.GetString (cursor.GetColumnIndex (Android.App.DownloadManager.ColumnUri));
         }
 
-        public void StartDownload (Android.App.DownloadManager downloadManager, string destinationPathName)
+        private bool _mobileNetworkAllowed = true;
+
+        public bool MobileNetworkAllowed
         {
-            using (var downloadUrl = Uri.Parse (Url))
-            using (var request = new Android.App.DownloadManager.Request (downloadUrl)) {
+            get
+            {
+                return _mobileNetworkAllowed;
+            }
+            set
+            {
+                _mobileNetworkAllowed = value;
+                if (Request != null && value == false) {
+                    Request.SetAllowedNetworkTypes(Android.App.DownloadNetwork.Wifi);
```

Comment:
The sample has that set here: https://github.com/SimonSimCity/Xamarin-CrossDownloadManager/blob/master/Sample/Droid/DownloadExample.Droid.csproj#L18
But also has it static set to 6.0 at the same time: https://github.com/SimonSimCity/Xamarin-CrossDownloadManager/blob/master/Sample/Droid/DownloadExample.Droid.csproj#L11